README.md

```diff
@@ -1,33 +1,50 @@
 
 ## Getting Started
 
-You can preview the live CI workflow for this application [on CircleCI](https://app.circleci.com/pipelines/github/dsayling/sample-flask?branch=main).
+You can preview the live CI workflow for this application [on CircleCI](https://app.circleci.com/pipelines/github/CircleCI-Public/sample-javascript-cfd?branch=main).
+
+The [CircleCI configuration](https://circleci.com/docs/2.0/configuration-reference/) file can be found at [.circleci/config.yml](https://github.com/CircleCI-Public/sample-javascript-cfd/blob/master/.circleci/config.yml).
+
+## About This App
+
+This sample application is a _Single Page Application_ written in [Vue.js](https://github.com/vuejs/vue), a popular front-end JavaScript framework, and utilizes the [Ionic Framework](https://ionicframework.com/) which will allow us to deploy our application to mobile devices.
+
+_Both [Vue.js](https://github.com/vuejs/vue) and [Ionic Framework](https://ionicframework.com/) are also users of CircleCI 🎉, and you can view the CI configs they use in their respective repositories_
 
-The [CircleCI configuration](https://circleci.com/docs/2.0/configuration-reference/) file can be found at [.circleci/config.yml](https://github.com/dsayling/sample-flask/blob/main/.circleci/config.yml).
+### Continuous Food Delivery
 
+When you load up the app, you will be presented with a menu from the _Continuous Food Delivery_ service, and a cart for items you wish to order. Click on items from the menu to have them added to your cart in real-time and the order total will automatically calculate.
 
-## Build and Test Locally
+![Preview Gif of Continuous Food Delivery app](https://github.com/CircleCI-Public/sample-javascript-cfd/blob/master/.github/img/preview.gif?raw=true)
 
-If you would like to try this application out locally, you can find runtime instructions below.
+### API Server
+
+CFD(Continuous Food Delivery) is a sample application which relies on an API server to both fetch menu items, as well as add them to your cart. If you would like to run this project locally, you must supply a valid CFD API service, such as one of the following sample projects for back-end languages:
+
+| Language |  GitHub | Description |
+|---|---|---|
+|  Python | [Link](https://github.com/CircleCI-Public/sample-python-cfd)  | A Python-Flask CFD API server  |
+
+## Run and Test Locally
 
 ### Requirements
 
 Node v13.11+
 
 ### Run Local Dev Server
 
-Start the local development server with the following command:
+1. Begin first by selecting an appropriate CFD API service from the list above and follow the repo's instructions for launching the API service.
+
+2. Start the local development server with the following command:
 
 ```
 ionic serve
```

Comment:
do you need to run npm install first?

Reply:
You do but for running this locally im assuming you know to do this. You will also be told this in an error if you dont.